azure-communication-ui/demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/CallCompositeManager.kt

```diff
@@ -719,6 +719,9 @@ class CallCompositeManager(private val context: Context) {
         }
         if (!SettingsFeatures.getCallScreenInformationTitle().isNullOrEmpty() ||
             !SettingsFeatures.getCallScreenInformationSubtitle().isNullOrEmpty() ||
+            /* <CALL_START_TIME> */
+            SettingsFeatures.getCallScreenShowCallDuration() ||
```

Comment:
This will result in not testing  `SettingsFeatures.getCallScreenShowCallDuration()` == false
We need to separate all three value title, subtitle and timer.